Realm.Shared/linq/RealmResultsVisitor.cs

```diff
@@ -117,6 +118,29 @@ private void AddSort(LambdaExpression lambda, bool isStarting, bool ascending)
         }
 
 
+        private RowHandle VisitElementAt(MethodCallExpression m)
+        {
+            Visit(m.Arguments.First());
+            var index = (int)ExtractConstantValue(m.Arguments.Last());
+
+            RowHandle row = null;
```

Comment:
I am pretty sure the compiler was (wrongly) complaining about using an uninitialised variable.